frontend/src/lib/lemon-ui/LemonProgress/LemonProgress.tsx

```diff
@@ -0,0 +1,40 @@
+import './LemonProgress.scss'
+
+import clsx from 'clsx'
+
+export type LemonProgressProps = {
+    size?: 'medium' | 'large'
+    trackColor?: string
+    percent: number
+    children?: React.ReactNode
+    className?: string
+}
+
+export const LemonProgress = ({
```

Comment:
Now that I think about this component, it would actually totally make sense to use it as the base of `LemonSlider`, just without the transition 🤔 I might do this in another PR